src/features/tags/hooks/useTagMutation.ts

```diff
@@ -3,17 +3,18 @@ import { tagAssigned, tagUnassigned } from '../store';
 import { useApiClient, useAppDispatch } from 'core/hooks';
 
 interface UseTagMutationReturn {
-  assignToPerson: (personId: number, value?: string) => void;
-  removeFromPerson: (personId: number) => Promise<void>;
+  assignToPerson: (personId: number, tagId: number, value?: string) => void;
+  removeFromPerson: (personId: number, tagId: number) => Promise<void>;
 }
-export default function useTagMutation(
-  orgId: number,
-  tagId: number
-): UseTagMutationReturn {
+export default function useTagMutation(orgId: number): UseTagMutationReturn {
```

Comment:
I don't really think of this as a "tag mutation", because it doesn't mutate (change) tags. I realize the source of this hook is not this PR, so I'm not suggesting we change it now, but I would like to share my thoughts and we can maybe approach a more common idea of how to name the hooks.

A "mutation" is an act which changes an object. A "tag mutation" to me sounds like something that would change a tag.

What does it mean to "assign a tag to a person"? Is it changing the tag? In my opinion, it is not. I might accept the notion that it is changing the person, but in a more strict sense, it isn't changing either the person or the tag. It is _connecting_ (or _attaching_, _applying_) the tag to the person. We generally use the term "tagging" to mean "attaching a tag to".

So a more suitable name for this hook, in my opinion, is something like `usePersonTagging`, or if we want to use a single hook for all tagging operations regardless of what we're tagging (a person, journey instance, location or other thing) we might call it `useTagging()`.

I would also consider `useTaggingMutations()`, which expresses the similarities of the functions returned by this hook to those returned by other mutation hooks, while not implying that it's mutating some particular object (but rather the "tagging" state in the backend database, generally).